### PR TITLE
Feature/25 correct style sheet handling

### PR DESCRIPTION
**Pull request by kpurrmann**
See comment on #25

### CODE REVIEW

File: Classes/View/PdfView.php [hunk 1]
```diff
@@ -127,7 +127,21 @@ private function replaceStrings($content) {
...
+                if (!empty($href[2]) &&
+                        ($href[1] === '/' . str_replace('&', '&amp;', GeneralUtility::getIndpEnv(TYPO3_SITE_SCRIPT)))
+                ) {
+                    return 'href="#' . $href[2] . '"';
+                }
+            }, $completeATag);
+
+            return $returnTag;
+
+        }, $content);
```
Comment (martin-helmich) [inline Classes/View/PdfView.php]: :+1: on the other changes, but I think I'd prefer a separate pull request for the link-handling stuff.

File: Classes/View/PdfView.php [hunk 2]
```diff
@@ -148,7 +162,12 @@ protected function getPdfObject() {
         /* @var $pdf \mPDF */
         $pdf = $this->objectManager->get('mPDF', '', $pageFormat . '-' . $pageOrientation);
         $pdf->SetMargins($leftMargin, $rightMargin, $topMargin);
-        $pdf->CSSselectMedia = $styleSheet;
+
+        if ($styleSheet == 'print' || $styleSheet == 'screen') {
```
Comment (martin-helmich) [inline Classes/View/PdfView.php]: There's still a default value for `$styleSheet` in line 160. Should that be changed, too?

Comment (kpurrmann) [inline Classes/View/PdfView.php]: It is not needed

**Comment by kpurrmann** [comment]
I will split to several pull requests

File: Classes/View/PdfView.php (hunk 2 above)
Comment (martin-helmich) [inline Classes/View/PdfView.php]: :+1: Allright.

**Pull request closed without merging**